api/src/main/java/org/openmrs/module/m2sysbiometrics/bioplugin/AbstractBioServerClient.java

```diff
@@ -108,6 +112,13 @@ public String delete(String subjectId) {
         return response.getDeleteIDResult();
     }
 
+    @Override
+    public boolean isFingerScanExists(M2SysCaptureResponse scannedFingers) {
```

Comment:
Lets move this response parsing a level higher, not the bioserver client.

Reply:
Ok, I will move it to M2SysV105Client